internal/pb/piece.pb.go

```diff
@@ -0,0 +1,164 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
it doesn't make sense to push this file to git as far as you are compiling it from .proto files in CI. No?

Reply:
i see. Then you can remove 'compile proto' steps from CI?

Reply:
Yes, I think we don't need this step. Removed